### ISSUE TITLE
Minimize and Maximized buttons gets mis-aligned by 1 pixel when a window is in fullscreen

### ISSUE THREAD

**Issue by aragubas** [kupiqu/SierraBreezeEnhanced#119]
Its kinda hard to see but it annoys me so much, when a window is in fullscreen and the window buttons are on the left, the minimize and maximize buttons get shifted 1 pixel to the right its so weird 

A normal window, with all the buttons in the correct place
![image](https://user-images.githubusercontent.com/47148297/229298738-bd1ef108-ef65-4253-990d-efa8d703d2d3.png)

A maximized window, with the buttons mis-aligned by 1 pixel

![image](https://user-images.githubusercontent.com/47148297/229298740-ea4aa5a8-14d9-48e2-bd8f-b3d1dcfd104d.png)

I made this recording overlaying the two screenshots, I snapped the windows into the top left corner so that the window title bars align. The VSCode window is the window in fullscreen and that other window that I'm changing the opacity is the normal window. The buttons kind of shift around xP

https://user-images.githubusercontent.com/47148297/229299212-5cefebdd-4f0d-41d9-9a1e-ace831c3e8b8.mp4

**Comment by kupiqu** [comment]
Hey, nice effect 😁

I'm assuming you're using the last release or master version, right?

**Comment by kupiqu** [comment]
Would you mind sharing a screenshot of your settings for me to try reproducing?

**Comment by aragubas** [comment]
> Hey, nice effect grin
> 
> I'm assuming you're using the last release or master version, right?

I compiled from the master branch!

**Comment by aragubas** [comment]
> Would you mind sharing a screenshot of your settings for me to try reproducing?

Sure! here it is! 
![image](https://user-images.githubusercontent.com/47148297/229305211-4442b920-a2ae-49ad-8195-42a2e0cfc72d.png)
![image](https://user-images.githubusercontent.com/47148297/229305224-03a6a557-5445-4edd-a202-92db532f28a1.png)

**Comment by kupiqu** [comment]
I cannot reproduce. I use the same window top-left and click green button several times (maximize, restore, maximize...), and nothing moves. Can you try that? Do you see the issue in all your applications?

**Comment by aragubas** [comment]
> I cannot reproduce. I use the same window top-left and click green button several times (maximize, restore, maximize...), and nothing moves. Can you try that? Do you see the issue in all your applications?

Yes I see this issue in all applications (except for GTK since it uses a different theme)

**Comment by aragubas** [comment]
Ah I forgot to mention, but I'm in KDE Neon running `plasmashell 5.27.3` and `kwin 5.27.3`

**Comment by kupiqu** [comment]
I also use kde neon, same version, but cannot reproduce your issue.

**Comment by kupiqu** [comment]
You could check whether this is still an issue in version 2.0.0

**Comment by aragubas** [comment]
Alright! I'll test

**Comment by aragubas** [comment]
was the project ported to KDE 6 at some point, I'm on Fedora KDE and I don't have any KF6 packages

**Comment by kupiqu** [comment]
Yes, it is ported to plasma 6
